### ISSUE TITLE
Use Alphine WSL2 image to boot the Guix system.

### ISSUE THREAD

**Issue by cireu** [giuliano108/guix-packages#2]
Thanks for your Guix-on-WSL2.md!

I found a much easier way to start a environment ready for bootstrap Guix. Use the Alpine Image provided in Microsoft Store.
It's very small (about 2 MiB) and provide anything needed for bootstrap. I think this is much simpler than using Busybox rootfs and do the rest job manually.

**Comment by cireu** [comment]
Alphine -> Alpine

**Comment by giuliano108** [comment]
You are right, if you're interested in Guix as a "package manager", starting from Alpine will get you up and running much quicker than the procedure described in `Guix-on-WSL2.md`.

If you're interested in the [Guix System](https://en.wikipedia.org/wiki/GNU_Guix_System) (that is the Guix package manager plus the distribution built around it), `Guix-on-WSL2.md` is still relevant in my opinion.

**Comment by cireu** [comment]
> Guix-on-WSL2.md is still relevant in my opinion.

The post binary install section(How to make a WSL compatible `operating-system` record and boot it) of this article is very useful. But I think user can use Alpine when they're not ready for the Guix binary installation, because it's simpler than the manual way(boot Busybox rootfs, mount device, write etc files) described in `Guix-on-WSL2.md`.

Maybe you can add I tips to tell readers that using Alpine to do binary installation is another possible approach(Then they can use `guix system reconfigure` to override the original Alpine distro)
